guidelines/chair-group.md

```diff
@@ -0,0 +1,35 @@
+## Chair group
+
+#### Definition and Motivation
+
+The Chair Group is a representative body of the Message Format WG that consists in spreading of the WG responsibilities among WG members that volunteered to get involved in the group chairing activities. Being part of a chair group requires an additional commitment that comes with a set of duties/responsibilities.
+
+#### Responsibilities
+
+- Ensure that the composition of the WG is well balanced in order that a high number of representatives are involved in the work of WG.
+
+- Encourage the participants, attending the meeting and to collaborate actively.
+
+- Manage and accept WG applications.
+
+- Manage and organize all WG communication channels, email, mail group, boards and Github repositories
+  
+- Prioritize, label and organize the tasks of the WG.
+
```

Comment:
New bullet suggestion:

"Formulate concrete questions that the WG can answer in their monthly meetings."